core/transformation/pathfinders/round_1/transform.py

```diff
@@ -164,17 +164,19 @@ def _project_details(
     organisation_name = df_dict["Organisation name"].iloc[0, 0]
     programme_id = programme_name_to_id_mapping[organisation_name]
     project_ids = df_dict["Project location"]["Project name"].map(project_name_to_id_mapping)
-    location_multiplicities = df_dict["Project location"]['Project full postcode/postcodes (e.g., "AB1D 2EF")'].map(
-        lambda x: "Multiple" if "," in x else "Single"
+    location_multiplicities = df_dict["Project location"][
```

Comment:
so we currently create this field during ingest? I'm not sure we need to do this, we only captured it in Towns Fund as it was on the ingest spreadsheet. The answer to this can be inferred from the Data model via the m2m relationship with geospatial (once it's implemented) or the number of postcodes that get stored in the "postcodes" array field.

Was there a requirement to calculate this field for Pathfinders? If not I would just remove `location_multiplicities`

Reply:
It's a potentially misleading field originally intended to capture user entered data

Reply:
Have just removed this now